_field-types/mapping-parameters/index-parameter.md

```diff
@@ -173,3 +196,47 @@ The following error response indicates that the search query failed because the
   "status": 400
 }
 ```
+
+For text fields, the index parameter being false is enough to disable searching on that field (this is because text fields do not support doc_values), however, for the other field types, we must also disable `doc_values` to have the same effect.  
+
+Query `products-no-index` using the `name` field:
+
+```json
+POST /products-no-index/_search
+{
+  "query": {
+    "term": {
+      "name": {
+        "value": "doc1"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The following response indicates that the search query succeeded because the name field is not indexed but still has `doc_values` enabled:
```

Comment:
```suggestion
The following response confirms that the search query succeeded because the `name` field, though not indexed, has `doc_values` enabled:
```